SCons/Script/__init__.py

```diff
@@ -30,16 +30,16 @@
 some other module.  If it's specific to the "scons" script invocation,
 it goes here.
 """
-
```

Comment:
this section is just a sort, not a change. It will never be possible to make checkers entirely happy here, because there's some *code* that has to run before some imports to get things right. Didn't feel like tagging every single import following the code bits, just did one that's well separated from the general import section (see line 138)